core/src/main/java/org/lflang/federated/generator/FederateInstance.java

```diff
@@ -100,9 +100,9 @@ public FederateInstance(
     // If the instantiation is in a bank, then we have to append
     // the bank index to the name.
     if (instantiation.getWidthSpec() != null) {
-      this.name = "federate__" + instantiation.getName() + "__" + bankIndex;
+      this.name = instantiation.getName() + "__" + bankIndex;
```

Comment:
This should be fine because instance names are already guaranteed to be unique within a reactor (the top level, in this case).

Reply:
This seems to be breaking some tests in the C target because some `.lft` files have a "federate__" prefix. Alternative could be when copying the user specified `yml` file, add the prefix `federate__` to each of the service names. However this would introduce another dependency `snakeyaml`.

Reply:
I'm surprised that a change to service names for Docker Compose would break tracing. Isn't it just a test that needs to be updated? Tagging @edwardalee as he might be able to comment on this.

Reply:
There seems to be a test trace with name `a_0.lft` and another trace with `federate__a_0.lft` so simply removing the "federate__" from test files would not work. This change also changes the names of the generated `.lf` files for each federate, so this might cause the generated trace to have different names too. I think adding a prefix to each service in the `yml` file when copying should be a relatively simpler fix.

Reply:
Why not not just change `TargetConfigTests.java:87` to:
```
URI.createURI(tempDir.resolve("src/Federation.lf").toUri().toString()),
```

Reply:
I added this change and all tests have passed!